lib/player.rb

```diff
@@ -0,0 +1,11 @@
+class Player
```

Comment:
Let's make sure our `Player` class has:

- a way to store its moves
- a public method that represents making a move.

Reply:
As we made our conceptualization, we think Match is the proper class to take control of the moves.

Reply:
we made the public method for the player => make_move